Port 7533 Update wiz docs to show information on config params (#1071)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz.md

## Prerequisites

### Port Credentials

<FindCredentials />

### Wiz Credentials

You need the following connection details to configure Wiz:

- Wiz API URL (API Endpoint URL)
- Wiz Token URL
- Client ID and Client Secret

:::info Wiz Token URL
There are two possible endpoints depending on your service account's identity provider:

- Amazon Cognito:	https://auth.app.wiz.io/oauth/token
- Auth0:	https://auth.wiz.io/oauth/token

Learn more [here](https://win.wiz.io/reference/quickstart#generate-a-bearer-token-and-start-using-wiz-api).
:::

<br />

1. **Finding Your Wiz API URL**:

>     - Log in to Wiz account.

Reviewer: ```suggestion
    - Login to your Wiz account.
```

>     - Click the User Profile icon (top right) and select "User Settings."

Reviewer: ```suggestion
    - Click the **User Profile** icon available at the top right of the screen and click the **User Settings** option.
```

>     - Click "Tenant" in the left menu.

Reviewer: ```suggestion
    - Click the **Tenant** option from the left options menu.
```

>     - Copy the API Endpoint URL: It will be displayed on the screen.

Reviewer: ```suggestion
    - The system displays the **API Endpoint URL**.
    - Copy and save the **API URL** to use while configuring the Wiz intergration.
```

> <img src='/img/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wizApi.png' width='85%' border='1px' />

Reviewer: This image is wrong, take a better look at the docs @hadar-co linked, the image that should be here is different.

For more details, refer to the [documentation](https://docs.wiz.io/wiz-docs/docs/using-the-wiz-api#the-graphql-endpoint)

<br />

2. **Getting the Client ID and Client Secret**

You should use a Wiz account with Project Admin role to complete this step.

>     - Log in to Wiz: Access your Wiz account.

Reviewer: This section is missing one of the images from the guide @hadar-co linked

> You should use a Wiz account with the Project Admin role to complete this step.
>     - Log in to Wiz: Access your Wiz account.

Reviewer: ```suggestion
You must create a service account in Wiz to generate the Client ID and Client Secret. Follow the below steps to get the Client ID and Client Secret:
    - Login to **Wiz with the Project Admin role**.
```

>     - Open Settings: Click the Settings icon (top-right).

Reviewer: ```suggestion
    - Click the **Settings** icon available at the top-right of the page.
```

  <img src='/img/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wizAddSvcAccount.png' width='85%' border='1px' />

>     - Go to Service Accounts: In the left menu, find and select "Service Accounts".

Reviewer: ```suggestion
    - On the Settings page, Click **Service Accounts** from the left menu.
```

    - Create a Service Account:

>         - Click "Add Service Account".

Reviewer: ```suggestion
        - Click **Add Service Account**.
```

>         - Provide a descriptive "Service Account Name".

Reviewer: ```suggestion
        - Provide a descriptive **Service Account Name**.
```

>         - Type: Select "Custom Integration (GraphQL API)".

Reviewer: ```suggestion
        - Type: Select **Custom Integration (GraphQL API)**.
```

## Prerequisites

1.**Port Credentials**:

<FindCredentials />

2. **Wiz Credentials**:

> :::info GETTING YOUR WIZ CREDENTIALS

Reviewer 2: @phalbert sending an email in order to get your client ID and secret seems strange. Here is a doc page of another product that explains how to acquire them:
https://docs.safe.security/docs/wizio#42-client-id-and-client-secret

Please change the instructions to these.

Author: Thanks a lot for this link. I have revised the instructions now.
